app/util/getUserTokens.ts

```diff
@@ -0,0 +1,13 @@
+import { useAccount, useTokens } from "~app/hook/storage"
+import { getChainName } from "~packages/network/util"
+
+export const getUserTokens = () => {
```

Comment:
[future] I think maybe we could directly doing this in the useTokens storage hook. Could be considered in future PR.

Reply:
Sure! That will be more appropriate.